.github/workflows/packages.yaml

```diff
@@ -6,8 +6,13 @@ on:
   release:
     types: [released, prereleased]
   workflow_dispatch:
+permissions:
+  contents: read
+
 jobs:
   deb:
+    permissions:
+      contents: write  # for actions/upload-release-asset to upload release asset
```

Comment:
Is there documentation on what the different permission scopes actually correspond to? I'm confused as to the difference between `contents` and `packages` (it seems like the relevant permission here would be packages?)